pkg/image/validation/kubernetes_test.go

```diff
@@ -14,7 +18,47 @@ var validNetwork = image.Network{
 	APIVIP:  "127.0.0.1",
 }
 
+func setupContext(t *testing.T) (ctx *image.Context, teardown func()) {
```

Comment:
We're barely using any of these directories - can't we just create it inline?

Reply:
How about I greatly simplify the function?